### DISCUSSION TITLE
Null contribution returned after mmm.fit

### DISCUSSION THREAD

**Discussion by miaox018** [google/lightweight_mmm#132] [Q&A] [answered]
hey team, first of all, thank you so much for providing this product!

I ran the code successfully but the contribution returned all nan. I checked all the input data and there wasn't any null value. How can we understand the potential issues here? I ran mmm.trace["coef_media"] and got valid numbers. Thanks!

**Comment by michevan** [comment]
Hi @miaox018 ! I'm not sure offhand, but my first guess would be that you might have some negative numbers in your trace; if so, can you try dropping those channels and see if the plot works?

**Comment by miaox018** [comment, marked as answer]
Thanks! I actually found it later that one media variable has 0 spend during the modeling time period. So it's already resolved now.
